test/abstract_syntax_tree.coffee

```diff
@@ -219,40 +607,101 @@ test "AST as expected for ModuleSpecifierList node", ->
   # TODO
```

Comment:
I think `ModuleSpecifierList` is a parent class that would never be output on its own. If that’s correct, we shouldn’t need to test it (well, we couldn’t, actually).

Reply:
If it’s never output in the AST, we don’t need a TODO for it.